ankihub/gui/terms_dialog.py

```diff
@@ -0,0 +1,53 @@
+from typing import Any, Optional
+
+from aqt.qt import QColor, QUrl, QVBoxLayout
+from aqt.webview import AnkiWebPage, AnkiWebView
+
+from ..gui.webview import AlwaysOnTopOfParentDialog, AuthenticationRequestInterceptor
+from ..settings import config
+
+
+class TermsAndConditionsDialog(AlwaysOnTopOfParentDialog):
+    dialog: Optional["TermsAndConditionsDialog"] = None
+
+    def __init__(self, parent: Any = None) -> None:
+        super().__init__(parent)
+        self._setup_ui()
+
+    def _setup_ui(self) -> None:
+        self.setWindowTitle("Settings")
+        self.resize(1000, 800)
+
+        self.web = AnkiWebView(parent=self)
+        self.web.set_open_links_externally(False)
+        self.interceptor = AuthenticationRequestInterceptor(self.web)
+        page = AnkiWebPage(self.web._onBridgeCmd)
+        page.open_links_externally = False
+        self.web.setPage(page)
+        self.web.page().profile().setUrlRequestInterceptor(self.interceptor)
+        self.web.page().setBackgroundColor(QColor("white"))
+        self.web.page().setUrl(
+            QUrl(f"{config.app_url}/users/school-info-and-terms-and-conditions/")
+        )
+
+        layout = QVBoxLayout()
```

Comment:
:wrench: 
```suggestion
        layout = QVBoxLayout()
        layout.setContentsMargins(0, 0, 0, 0)
```

**Before**
<img src="https://github.com/user-attachments/assets/3f88f688-4867-4694-8c6e-e23e964b0eec" width="500" />

**After**
<img src="https://github.com/user-attachments/assets/2371a3e5-118c-43a1-a954-b978f85582cc" width="500" />